site/ristretto.md

```diff
@@ -41,9 +43,3 @@ This site is organized into several chapters:
 - [Explicit Formulas](./formulas/index.html) describes how to implement Ristretto.
 - [Test Vectors](./test_vectors/index.html) contains test vectors for the Ristretto functions.
 - [Ristretto Implementations](./implementations.html) contains a list of implementations of Ristretto.
-
-## TODO
-
-This is the landing page. It should link to the other pages, have the
-coffee graphic, and have the author list.
-
```

Comment:
I would prefer to keep this for now...

Reply:
Perhaps a `TODO.md` is a better place for it than the front page?

Reply:
I would have rather put it on the website, so that until the website is done, there's an indication that it's not finished yet.

Reply:
Could always choose from one of these http://www.textfiles.com/underconstruction/ 😜

How about a note at the top of the page that the site is a work in progress?